public/extensions/breakout.js

```diff
@@ -0,0 +1,116 @@
+// @ts-check
+
+import daily from "./core.js";
+import Socket from "./socketiostate.js";
+
+let call;
+let socket;
+
+// Initial state
+let state = {
+  initialRoomUrl: "",
+  breakoutStarted: false,
+  participants: [{ user_name: "", roomUrl: "" }],
+};
+
+daily.beforeCreateFrame((parentEl, properties) => {
+  if (!properties.customTrayButtons) {
```

Comment:
We might eventually want to limit this to room owners only, but I'm fine shipping as-is

Reply:
I thought the same thing, it's REALLY easy to build many features on top of breakout rooms when you start building one. Let's get the first chunk out the door before we get too sucked in though